pkg/syncer/sync.go

```diff
@@ -44,26 +43,25 @@ func pp(secret corev1.Secret, credentials config.Credentials, config k8s.Config,
 // Sync does the same thing as Inject, but by iterating over the existing k8s Secrets
 func Sync(config k8s.Config, namespace string, credentials config.Credentials) error {
 	secretsClient, err := k8s.GetSecretsClient(config, namespace)
-
 	if err != nil {
 		return fmt.Errorf("[ERROR] error getting a Kubernetes Client API Secrets Client: %s", err)
 	}
-	log.Printf("[DEBUG] getting a list of Secrets in namespace '%s'", namespace)
+	fmt.Printf("[DEBUG] getting a list of Secrets in namespace '%s'", namespace)
```

Comment:
as discussed: if the intention is to be able to disable logs you can configure it globally using:
```
log.SetOutput(io.Discard)
```

Reply:
✅ reverted back to log output and used your suggestion. Thank you!